Taskfile.yml

```diff
@@ -223,18 +236,36 @@ tasks:
       - setup:ui
     cmds:
       - yarn generate
+    sources:
+      - "**"
+      - exclude: .nuxt/**
+      - exclude: .output/**
+      - exclude: dist/**
+      - exclude: node_modules/.cache/**
 
   ui:lint:
     desc: runs the frontend linter
     dir: frontend
     cmds:
       - yarn lint --max-warnings=0
+    sources:
+      - "**"
+      - exclude: .nuxt/**
+      - exclude: .output/**
+      - exclude: dist/**
+      - exclude: node_modules/.cache/**
```

Comment:
comment: the sources definitions are probably overly broad for some of these frontend tasks, but this works and should reduce some unnecessary re-runs.

The downside of too-broad sources lists is that `task` takes longer to compute the checksum for the change detection. 

Happy for other people to finetune these later.